UnityProject/Assets/UI/Fonts/Liliput_Steps/lilliput steps SDF.asset

```diff
@@ -2442,7 +2442,7 @@ Material:
     - _OutlineSoftness: 0
     - _OutlineUVSpeedX: 0
     - _OutlineUVSpeedY: 0
-    - _OutlineWidth: 0
+    - _OutlineWidth: 0.099
```

Comment:
owo what are these assets?

Reply:
Shadows for text because they look ugly without them